server/config/server.go

```diff
@@ -80,23 +82,18 @@ func (c *AppConfig) PostgresConnString() string {
 	defer c.mu.Unlock()
 
 	if postgresConnString := c.vp.GetString("postgresConnString"); postgresConnString != "" {
-		return postgresConnString
+		fmt.Println("ERROR: postgresConnString was discontinued. Migrate to the new postgres format")
```

Comment:
Something to think about in the future: we should move these server logs from `fmt` to something else, like `zap` or `logrus`, that we can centralize and configure later.
Does it make sense to create a ticket for that? (I can do that if needed!)

Reply:
+1, do you mind creating the ticket @danielbdias ?

Reply:
done! Issue: https://github.com/kubeshop/tracetest/issues/2996